### PR TITLE
use regular text, instead of hardcoded English images, for site title (fixes #140)

### LINKED ISSUE #140: "The Future of Web Games" heading should be real text instead of a graphic
Notice the heading at the top of https://games.mozilla.org/

- https://games.mozilla.org/assets/img/headline@2x.png
- https://games.mozilla.org/assets/img/headline_mobile.png

image

<hr>

Related issue: #123

### PR DESCRIPTION
**Pull request by cvan**
## extra-wide-desktop width

 

<hr>

## wide-desktop width

 

<hr>

## desktop width

 

<hr>

## narrow-desktop width

 

<hr>

## tablet width

 

<hr>

## mobile width

  

<hr>

## narrow-mobile width

 

<hr>

> **FYI:** the `Mozilla` wordmark text (set in the former typeface, FF Meta, for the `Mozilla` wordmark) in the bottom isn't rendered in the correct typeface because the TypeKit script included on the page is not whitelisted for the `localhost` origin, so I can't test that locally.

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
     <link rel="icon" type="image/png" sizes="196x196" href="assets/img/favicon-196x196.png">
     <link rel="shortcut icon" href="/favicon.ico">
     <link rel="apple-touch-icon" href="assets/img/apple-touch-icon.png">
-    <script src="https://use.typekit.net/glx7slj.js"></script>
-    <script>try{Typekit.load({ async: true });}catch(e){}</script>
+    <script src="https://use.typekit.net/glx7slj.js" defer></script>
```
Comment (cvan) [inline public_html/index.html]: this (and the verbose `html[wf-*]` styles above) make sure that loading the web-fonts from TypeKit doesn't block rendering of the page. it previously did. big perf improvement! 👍

File: public_html/index.html [hunk 2]
```diff
@@ -26,8 +26,7 @@
     </div>
     <div class="header">
       <div class="container">
-        <img class="headline headline-desktop" src="assets/img/headline@2x.png" alt="The Future of Web Games">
```
Comment (cvan) [inline public_html/index.html]: so happy 😄

Comment (michaelic) [reply public_html/index.html]: ❤️  it's a beautiful thing for sure

File: public_html/assets/css/global/style.css [hunk 3]
```diff
@@ -283,40 +295,47 @@ html[dir="rtl"] #tabzilla {
...
+  font-weight: 300;
+  letter-spacing: -.03ch;
+  line-height: 1;
+  margin: 0 auto;
   max-width: 90%;
 }
 
-.header img.headline-mobile {
-  display: none;
+@supports (-webkit-text-fill-color: transparent) {
```
Comment (michaelic) [inline public_html/assets/css/global/style.css]: I hadn't really seen any feature queries in the wild yet, this is neat

Comment (cvan) [reply public_html/assets/css/global/style.css]: I didn't know about them until I started searching online for examples of this property. I didn't know `@supports` had shipped yet! and it's just safely skipped in older browsers 👍

**Comment by cvan** [comment]
thanks!

**Pull request merged**